app/forms/state_file/id_sales_use_tax_form.rb

```diff
@@ -0,0 +1,21 @@
+module StateFile
+  class IdSalesUseTaxForm < QuestionsForm
+    set_attributes_for :intake, :has_unpaid_sales_use_tax, :total_purchase_amount
+
+    validates :has_unpaid_sales_use_tax, inclusion: { in: %w[yes no], message: :blank }
+    validates :total_purchase_amount,
+      presence: true,
+      numericality: {
+        allow_blank: true,
+        greater_than_or_equal_to: 0,
+        message: I18n.t("state_file.questions.nc_sales_use_tax.edit.not_a_number")
```

Comment:
This i18n key says `nc_sales_use_tax`, I'd like to either see a different key created for id, or this one be made generic (I don't have a strong preference, but fewer keys is cheaper if that works out)